clang/include/clang/Format/Format.h

```diff
@@ -1010,7 +1010,8 @@ struct FormatStyle {
   /// \version 3.7
   DefinitionReturnTypeBreakingStyle AlwaysBreakAfterDefinitionReturnType;
 
-  /// The function declaration return type breaking style to use.
+  /// The function declaration return type breaking style to use.  This
+  /// option is **deprecated** and is retained for backwards compatibility.
   /// \version 3.8
   ReturnTypeBreakingStyle AlwaysBreakAfterReturnType;
```

Comment:
The actual member has to go.

Reply:
+1, but see https://github.com/llvm/llvm-project/pull/80827#issuecomment-1935386091.